docs/rules/space-before-blocks.md

```diff
@@ -94,7 +94,7 @@ for (;;) {
 try {} catch(a) {}
 ```
 
-The following patterns are not considered problems:
+Examples of **correct** code for this rule with the "never" option.
```

Comment:
You end with a colon on line 30 and a period here. I like the colon better, mind switching?